File: app-java/src/main/java/com/TritiumGaming/phasmophobiaevidencepicker/firebase/firestore/transactions/store/microtransactions/FirestoreMicrotransactions.java
package com.TritiumGaming.phasmophobiaevidencepicker.firebase.firestore.transactions.store.microtransactions;

import androidx.annotation.NonNull;

import com.TritiumGaming.phasmophobiaevidencepicker.firebase.firestore.transactions.store.FirestoreMarketplace;
import com.google.firebase.firestore.DocumentReference;

public class FirestoreMicrotransactions {

    private static final String
            DOCUMENT_MICROTRANSACTIONS = "Microtransactions";

    @NonNull
    public static DocumentReference getMicroTransactionsDocument() {
        return FirestoreMarketplace.getStoreCollection()
                .document(DOCUMENT_MICROTRANSACTIONS);
    }

}

File: app-compose/src/main/java/com/tritiumgaming/phasmophobiaevidencepicker/firebase/firestore/transactions/store/microtransactions/FirestoreMicrotransactions.kt
package com.tritiumgaming.phasmophobiaevidencepicker.firebase.firestore.transactions.store.microtransactions

import com.google.firebase.firestore.DocumentReference
import com.tritiumgaming.phasmophobiaevidencepicker.firebase.firestore.transactions.store.FirestoreMarketplace.Companion.storeCollection

class FirestoreMicrotransactions {
    companion object {
        private const val DOCUMENT_MICRO_TRANSACTIONS = "Microtransactions"

        val microTransactionsDocument: DocumentReference
            get() = storeCollection
                .document(DOCUMENT_MICRO_TRANSACTIONS)
    }

}
